compareECLFiles.cmake

```diff
@@ -758,6 +758,13 @@ add_test_compare_restarted_simulation(CASENAME spe9
                                       REL_TOL ${rel_tol_restart}
                                       TEST_ARGS --sched-restart=false)
 
+add_test_compare_restarted_simulation(CASENAME model4
+                                      FILENAME MOD4_GRP
+                                      SIMULATOR flow
+                                      ABS_TOL ${abs_tol_restart}
+                                      REL_TOL ${rel_tol_restart}
+                                      TEST_ARGS --sched-restart=True)
```

Comment:
Does the property system supports an uppercase `T` in `True` here?

Reply:
Yes - I am quite sure it does.

Reply:
>> Does the property system support an uppercase `T` in `True` here?
>
> Yes - I am quite sure it does.

Yes, you're right.  The ultimate arbiter of these values is `Dune::ParameterTree::Parser<bool>` which does (a rough version of) [case-insensitive string matching](https://gitlab.dune-project.org/core/dune-common/-/blob/4f293a61edd0ae4cc29d510243a068a49907cab7/dune/common/parametertree.hh#L294) of the parameter values.